docs/kratos/social-signin/30_apple.mdx

```diff
@@ -16,22 +16,25 @@ one.
 
 ## Webbrowser
 
-````mdx-code-block
+```mdx-code-block
 import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs>
 <TabItem value="console" label="Ory Console" default>
+```
 
 Follow these steps to add Apple as a social sign-in provider to your project using the Ory Console:
 
 1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
 2. Click the switch next to the Apple logo to start the configuration.
 3. Copy the Redirect URI and save it for later use.
 4. Using an Apple Developer Account, create an app, a service, and a private key:
-   1. To set up an application, navigate to the Apple Developer Dashboard and go to [Certs, Identifiers, and Profiles then Identifiers](https://developer.apple.com/account/resources/identifiers/list).
-   2. Create a new [**App IDs** identifier](https://developer.apple.com/account/resources/identifiers/add/bundleId). When prompted for a type select **App**.
+   1. To set up an application, navigate to the Apple Developer Dashboard and go to
```

Comment:
is it possible now with docusaurus v3 to respect ordered lists? 
it renders like this. : 
<img width="953" alt="Screenshot 2024-06-10 at 12 37 54" src="https://github.com/ory/docs/assets/17701416/38366487-2a85-4108-ac72-c4137c5a5e5b">

Reply:
> to respect ordered lists

What do you mean by this? It's a numbered ordered list, but the second level is using roman numerals, to make the distinction.

Reply:
yeah, I was thinking of not using roman numerals

Reply:
I don't really see anything in regard to that. But we can probably break up this section to not have a nested list in there.